test/integration/framework/etcd.go

```diff
@@ -136,6 +140,27 @@ func startEtcd() (func(), error) {
 	if err := cmd.Start(); err != nil {
 		return nil, fmt.Errorf("failed to run etcd: %v", err)
 	}
+
+	var i int32 = 1
+
+	for i <= pollCount {
```

Comment:
You can define this const inside this method, which is more appropriate as its local to this functon

Reply:
Done.